src/EnergyPlus/DaylightingManager.cc

```diff
@@ -567,7 +567,8 @@ void CalcDayltgCoefficients(EnergyPlusData &state)
 
     if (state.dataDaylightingManager->doSkyReporting) {
         if (!state.dataGlobal->KickOffSizing && !state.dataGlobal->KickOffSimulation) {
-            if (state.dataDaylightingManager->FirstTimeDaylFacCalc && state.dataDaylightingManager->TotWindowsWithDayl > 0) {
+            if (state.dataDaylightingManager->FirstTimeDaylFacCalc && state.dataDaylightingManager->TotWindowsWithDayl > 0 &&
+                (!state.dataSysVars->DetailedSolarTimestepIntegration || state.dataGlobal->HourOfDay == 12)) {
```

Comment:
I'm really not sure about this change. I had 2 tests that were failing when DetailedSolarTimestepIntegration is true, because it tried to report directly at hourOfDay=1 the value at noon before it's computed.

Reply:
I'm not sure about this one either.  Daylighting.... @mjwitte or someone care to peek here?

Reply:
`&& (!state.dataSysVars->DetailedSolarTimestepIntegration || state.dataGlobal->HourOfDay == 12))`

Not a fan of this change. Let me experiment with a different fix.

Reply:
> `&& (!state.dataSysVars->DetailedSolarTimestepIntegration || state.dataGlobal->HourOfDay == 12))`
> 
> Not a fan of this change. Let me experiment with a different fix.

Never mind, this fix is appropriate. This is not just a unit test problem. Test file ReportDaylightFactors with ShadowCalculation changed to Timestep fails in a debug build (develop) in the same place, because (as @jmarrec had noted in the slack thread) the hour 12 values are not calculated yet with `state.dataSysVars->DetailedSolarTimestepIntegration`.

Reply:
And just for fun, here's the output with a release build of develop. I suppose we could post an issue that daylight factors don't report properly with timestep shadow calcs, but clearly nobody has tried that (ever) and it will be fixed with this PR.
```
! <Sky Daylight Factors>, Sky Type, MonthAndDay, Daylighting Control Name, Enclosure Name, Window Name, Reference Point, Daylight Factor
 Sky Daylight Factors,Clear Sky,01/21,PERIMETER_BOT_ZN_3_DAYLCTRL,PERIMETER_BOT_ZN_3,PERIMETER_BOT_ZN_3_WALL_NORTH_WINDOW,PERIMETER_BOT_ZN_3_DAYLREFPT1,3533895657667489933270916018909613907062005101881119392795255436020002031281589176835540497970064983355629857203380701242230589801061049128102347174951675390368240560046995087476935261441338275438186055107562997442898612530753270056168170023026688.
 Sky Daylight Factors,Clear Turbid Sky,01/21,PERIMETER_BOT_ZN_3_DAYLCTRL,PERIMETER_BOT_ZN_3,PERIMETER_BOT_ZN_3_WALL_NORTH_WINDOW,PERIMETER_BOT_ZN_3_DAYLREFPT1,2.1804E-153
 Sky Daylight Factors,Intermediate Sky,01/21,PERIMETER_BOT_ZN_3_DAYLCTRL,PERIMETER_BOT_ZN_3,PERIMETER_BOT_ZN_3_WALL_NORTH_WINDOW,PERIMETER_BOT_ZN_3_DAYLREFPT1,142963876623584389593035522072177012584467539105698209034063362592849524678467934174027789837564869347911808042753712029541366450090039049390973994416024325127201309174616442724104976719023587010143647066044010391991907957530046502039898435434315776.
```